j2735_msgs/msg/ControlMessage.msg

```diff
@@ -0,0 +1,114 @@
+#
+# ControlMessage.msg
+#
+# J2735 2016 message format.
+#
+# @author Mae Fromm
+# @version 0.1
+#
+# Description
+# ...
+
+# ControlMessage ::= IMPLICIT SEQUENCE {
+# 	version IA5String,
+# 	id OCTET STRING (SIZE(16)),
+# 	updated INTEGER (0..9223372036854775807),
+# 	vtypes SET OF VType,
+# 	schedule Schedule,
+# 	regulatory BOOLEAN,
+# 	controltype ControlType,
+# 	controlvalue ControlValue OPTIONAL,
+# 	pathParts INTEGER (0..127),
+# 	proj IA5String,
+# 	datum IA5String,
+# 	time INTEGER (0..9223372036854775807),
+# 	lon INTEGER (-1800000000..1799999999),
+# 	lat INTEGER (-900000000..900000000),
+# 	alt INTEGER (0..92760), -- decimeters offset by -4280, which is lowest elevation at Dead Sea, max value is mount everest
+# 	heading INTEGER (0..3599), -- tenths of degrees
+# 	points SEQUENCE (SIZE(2..100)) OF Point
+# }
+
+# Version ::= IA5String (SIZE(36))
+string version 
+
+# Id ::= OCTET STRING (SIZE(16))
+uint8[] id 
```

Comment:
I think you can make this and the other id (in a different message) fixed length arrays of size 16
uint8[16]
I believe the whole thing will always be filled so its good to enforce the length.